### PR TITLE
Fix wrong cmd of clang-offload-bundler

### PR DESCRIPTION
**Pull request by tomsang**
Fix code obj file generation failure of the following tests,
test_find_db, test_handle_test, test_reduce_test

### CODE REVIEW

File: src/hip/hip_build_utils.cpp [hunk 1]
```diff
@@ -193,7 +193,7 @@ boost::filesystem::path HipBuild(boost::optional<TmpDir>& tmp_dir,
 
         // call clang-offload-bundler
         tmp_dir->Execute(MIOPEN_OFFLOADBUNDLER_BIN,
-                         "--type=o --targets=hip-amdgcn-amd-amdhsa-" + dev_name + " --inputs=" +
+                         "--type=o --targets=hip-amdgcn-amd-amdhsa--" + dev_name + " --inputs=" +
```
Comment (pfultz2) [inline src/hip/hip_build_utils.cpp:196]: Is this change backwards compatible?

Comment (daniellowell) [reply src/hip/hip_build_utils.cpp:196]: Yeah IDK, good question.

Comment (yxsamliu) [reply src/hip/hip_build_utils.cpp:196]: backward compatibility means old executable can run on new runtime. we do not guarantee new executable can run on old runtime. Does MIOpen develop branch promise to work on all ROCm release?

one strange thing is that how could this issue happen? the compiler change passes ePSDB which has MIOpen test. Does this issue only happen to develop branch of this repo?

Comment (atamazov) [reply src/hip/hip_build_utils.cpp:196]: This is already implemented in #674, which provides backward compatibility.

**Comment by atamazov** [comment]
Superseded by #674

**Comment by atamazov** [comment]
@tomsang 
> Fix code obj file generation failure of the following tests,
> test_find_db, test_handle_test, test_reduce_test

Please let me know where do you observe these errors (in other words, how to reproduce them) and I'll make sure that #674 fixes them.

**Pull request closed without merging**